core/txpool/txpool.go

```diff
@@ -891,18 +909,26 @@ func (pool *TxPool) add(tx *types.Transaction, local bool) (replaced bool, err e
 	return replaced, nil
 }
 
-// isFuture reports whether the given transaction is immediately executable.
-func (pool *TxPool) isFuture(from common.Address, tx *types.Transaction) bool {
-	list := pool.pending[from]
-	if list == nil {
-		return pool.pendingNonces.get(from) != tx.Nonce()
+// isGapped reports whether the given transaction is immediately executable.
```

Comment:
Should this comment be the opposite? This reads that if this returns true the transaction is immediately executable, but it's vice versa after this change.

Reply:
This comment is as is from upstream, leaving it.